### DISCUSSION TITLE
Welcome to tosdr.org Discussions!

### DISCUSSION THREAD

**Discussion by JustinBack** [tosdr/tosdr.org#266] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KenEucker** [comment]
Cool!

Long time lurker and supporter of the Tos;DR project. Love all the work y'all do. 👏 👏

**Comment by ghost** [comment]
Hi! (crossposted this introduction to the #introduction channel on the Discord)

I don't exactly remember how I learned about ToS;DR, but it was probably through https://wikidata.org where I spend a fair amount of my time. I actually think me not remembering indicates how infrastructural the project is, it instantly made sense that it should exist once I found it, and had in some alternate world it not existed I imagine it only would be a matter of time before it was created.

Anyway, I think some of the best way to understand a (what ToS;DR calls a service) is sort of through the back door of the terms (well, it should be a front door given that it is supposed to be an upfront 'contract' before use, but so things go) -- showing what organisations are involved, expressing what interests they have in the 'service' and what perspectives they are therefore coming from, and so on. While I am certainly enthusiastic about the privacy-advocacy aspects of ToS;DR, I especially value it as a sort of scholarly resource, but one that aims to be available and useful to every netizen and internaut.
